addons/purchase_stock/models/purchase.py

```diff
@@ -453,7 +449,7 @@ def _prepare_stock_move_vals(self, picking, price_unit, product_uom_qty, product
             'picking_type_id': self.order_id.picking_type_id.id,
             'group_id': self.order_id.group_id.id,
             'origin': self.order_id.name,
-            'description_picking': description_picking,
+            'description_picking': self.name,
```

Comment:
To check with Thomas but I would add the line description to the description_picking

Reply:
@thomasdeleval Can I have your though on my last comment ? In case the product has a description picking for the receipts, this description is replaced by the vendor description
Thanks

Reply:
@Whenrow keep description_picking when not empty. If users want to see vendor ref and name, they can just add it in the picking description